Pod/Classes/Core/ARSwitchBoardModule.m

```diff
@@ -21,6 +21,11 @@ @implementation ARSwitchBoardModule
   [self invokeCallback:self.presentModalViewController reactTag:reactTag route:route];
 }
 
+RCT_EXPORT_METHOD(presentSearchViewController:(nonnull NSNumber *)reactTag)
```

Comment:
I know that all of these will have to be in the pod and not here; this is more of a sketch of what will need to happen later. All of this should be gone by the time this particular PR is mergeable.